website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -283,6 +283,7 @@ Be careful not to misuse or overuse the Generic ListItem. Relying on this escape
 Icons in ListItems are optional. Generally, we recommend letting the text speak for itself, but icons can add value in the following situations:
 
 - When they reinforce the content, e.g., `edit` for an edit or rename action. 
+- When using Interactive ListItems: using trailingIcons to indicate a link is external.
```

Comment:
Not overly opinionated on this, but I think this reads a bit better as it's more of a sentence.

```suggestion
- When using Interactive ListItems, a `trailingIcon` can indicate that a link is external.
```